styles.css

```diff
@@ -69,4 +69,16 @@
 
 .text-right{
     margin-top:  -2rem;
+}
+
+.right-ans{
+    background-color: rgb(0, 153, 0);
+}
+
+.wrong-ans{
+    background-color: rgb(201, 6, 6);
+}
+
+.ans-btn:focus{
+    background-color: #003973;
```

Comment:
Suggestion: use variable color naming, it will help you later, when you want to change color value

Reply:
sure, Thank you